### ISSUE TITLE
Windows build has execution error

### ISSUE THREAD

**Issue by sclayton33** [PistilliLab/CLAMSwrangler#9]
OS: Windows 10
CLAMS Wrangler version: 1.0.0

This issue only happens in the built executable, not in the native python version. It has been reproduced on two Windows systems.

~~~
Traceback (most recent call last):
  File "tkinter\__init__.py", line 1921, in __call__
  File "main.py", line 206, in main_process_clams_data
  File "clams_processing.py", line 129, in trim_all_clams_data
  File "main.py", line 29, in write
AttributeError: 'NoneType' object has no attribute 'write'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 395, in <module>
  File "tkinter\__init__.py", line 1458, in mainloop
  File "tkinter\__init__.py", line 1925, in __call__
  File "tkinter\__init__.py", line 1641, in _report_exception
  File "tkinter\__init__.py", line 2375, in report_callback_exception
  File "main.py", line 29, in write
AttributeError: 'NoneType' object has no attribute 'write'
~~~

**Comment by sclayton33** [comment]
Adding some debug statements shows there is a recursion error with the write function in Class StdoutRedirect

~~~
User
Traceback (most recent call last):
  File "tkinter\__init__.py", line 1921, in __call__
  File "main.py", line 215, in main_process_clams_data
  File "clams_processing.py", line 129, in trim_all_clams_data
  File "main.py", line 28, in write
  File "main.py", line 28, in write
  File "main.py", line 28, in write
  [Previous line repeated 494 more times]
RecursionError: maximum recursion depth exceeded while calling a Python object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 404, in <module>
  File "tkinter\__init__.py", line 1458, in mainloop
  File "tkinter\__init__.py", line 1925, in __call__
  File "tkinter\__init__.py", line 1641, in _report_exception
  File "tkinter\__init__.py", line 2375, in report_callback_exception
  File "main.py", line 28, in write
  File "main.py", line 28, in write
  File "main.py", line 28, in write
  [Previous line repeated 494 more times]
RecursionError: maximum recursion depth exceeded while calling a Python object
~~~

**Comment by sclayton33** [comment]
Fixed in commit [283b1a034ea7b1d5b7412765254d6ac261e33ff0](https://github.com/PistilliLab/CLAMSwrangler/commit/283b1a034ea7b1d5b7412765254d6ac261e33ff0).
